titus_isolate/crd/model/resource_usage_prediction.py

```diff
@@ -161,13 +162,21 @@ def __init__(self, raw: dict):
         self.prediction_ts_ms = int(raw.get(PREDICTION_TS_MS, '0'))
         self.metadata = raw.get(META_DATA, {})
         self.__predictions = {}
+        self.__pred_time2empty_batch = []
 
         preds = raw.get(PREDICTIONS)
         if preds is not None:
             for p in preds:
                 job_id = p.get(JOB_ID, "UNKNOWN_JOB_ID")
                 self.__predictions[job_id] = ResourceUsagePrediction(p)
 
+        if PRED_TIME2EMPTY_BATCH in self.metadata:
+            try:
+                self.__pred_time2empty_batch = [float(e) for e in self.metadata[PRED_TIME2EMPTY_BATCH].split(',')]
+            except:
+                pass
+            self.metadata.pop(PRED_TIME2EMPTY_BATCH)
```

Comment:
You're sure you don't want it in the metadata anymore for debugging?

Reply:
good point, moved to the `try` block